iac-extensions/docker/src/test/java/org/sonar/iac/docker/tree/impl/DockerImageImplTest.java

```diff
@@ -69,7 +69,7 @@ void imageWithInstructions() {
     assertThat(dockerImage.children()).hasExactlyElementsOfTypes(FromInstructionImpl.class, MaintainerInstructionImpl.class, ExposeInstructionImpl.class);
     assertThat(dockerImage.instructions()).hasSize(2);
     MaintainerInstruction maintainer = (MaintainerInstruction) dockerImage.instructions().get(0);
-    assertThat(maintainer.authors()).extracting(TextTree::value).containsExactly("bob");
+//    assertThat(maintainer.authors()).extracting(TextTree::value).containsExactly("bob");
```

Comment:
I would add a "todo" with ticket number to process this.

Reply:
Oh, this was not expected to be commented out. Nice catch.